packages/composite-checkout/src/lib/payment-methods/stripe-credit-card-fields.js

```diff
@@ -515,7 +515,7 @@ const LockIconGraphic = styled( LockIcon )`
 	height: 20px;
 `;
 
-function CardFieldIcon( { brand, isSummary } ) {
+export function CardFieldIcon( { brand, isSummary } ) {
```

Comment:
Since we're sharing this between payment methods, can we move this to its own file in `lib/payment-methods`?

Reply:
Good idea. I'll move it out.